### DISCUSSION TITLE
为 android 设备提供支持

### DISCUSSION THREAD

**Discussion by xxxxxliil** [mozillazg/ptcpdump#131] [Ideas]
在 android 设备上进行抓包确实可以利用 vpn 服务进行抓包，而且需求仅抓取特定进程（其实是 uid）时几乎是唯一选择。但是越来越多的 app 在 vpn 服务工作的情况下拒绝启动，这时候一个不需要 vpn 服务也能抓取特定进程的工具就很重要了

**Comment by mozillazg** [comment]
@xxxxxliil 感谢建议。不过这个需求的优先级可能暂时不会很高。我需要先买个二手安卓开发机，你有推荐的机型和购买渠道不？

**Reply by xxxxxliil** [reply]
例如说三星 s22u？想得到便宜价格可以看看韩版
